test/unit/plugins/vocablink.test.js

```diff
@@ -0,0 +1,42 @@
+const expect = require('expect');
+const parser = require('../../../src/cdoFlavoredParser');
+
+describe('vocablink', () => {
+  describe('render', () => {
+    it('cannot render vocablinks to html', () => {
+      const input = "[v some-word]";
+      const output = parser.sourceToHtml(input);
+      expect(output).toEqual("<p>[v some-word]</p>\n");
+    });
+  });
+
+  describe('redact', () => {
+    it('redacts vocablinks', () => {
+      const input = "[v some-word]";
+      const output = parser.sourceToRedacted(input);
+      expect(output).toEqual("[some-word][0]\n");
+    });
+
+    it('redacts vocablinks with word overrides', () => {
+      const input = "[v some-word][un-mot]";
+      const output = parser.sourceToRedacted(input);
+      expect(output).toEqual("[un-mot][0]\n");
```

Comment:
how is it that these overrides are used in practice -- is this the mechanism that lets us override the original vocab word with the one supplied by the translator?

Reply:
The overrides are actually brand-new syntax defined here https://github.com/mrjoshida/curriculumbuilder/pull/11 to support translations in the first place; so they are not used in source text in practice are are intended just to be that mechanism for translating, but I expect that now that the syntax is there it's gonna eventually be used